### DISCUSSION TITLE
(Question) Error when drawing Morgan Bits : AtomKekulizeException: non-ring atom 3 marked aromatic

### DISCUSSION THREAD

**Discussion by mohammad-saber** [rdkit/rdkit#3989] [Q&A]
I am trying to draw Morgan bits following the instruction [here](http://rdkit.blogspot.com/2018/10/using-new-fingerprint-bit-rendering-code.html). 

I am using the exact code in the article but the following error comes up:

`AtomKekulizeException: non-ring atom 3 marked aromatic
`

Code: 

```
from rdkit import Chem
from rdkit.Chem import rdMolDescriptors
from rdkit.Chem.Draw import IPythonConsole
from rdkit.Chem import Draw

epinephrine = Chem.MolFromSmiles('CNC[C@H](O)c1ccc(O)c(O)c1')

bi = {}
fp = rdMolDescriptors.GetMorganFingerprintAsBitVect(epinephrine, radius=2, bitInfo=bi)

tpls = [(epinephrine,x,bi) for x in fp.GetOnBits()]
Draw.DrawMorganBits(tpls[:12],molsPerRow=4,legends=[str(x) for x in fp.GetOnBits()][:12])
```

I couldn't understand the reason of this error. I really appreciate it if you could guide me on this.
I am using RDKit version : '2020.09.1'

**Comment by greglandrum** [comment]
This is due to a bug in the RDKit, it's discussed here:
https://github.com/rdkit/rdkit/issues/3572
There's a workaround described there, or you can install a newer version of the RDKit

**Reply by mohammad-saber** [reply]
Is it possible to update the current version instead of uninstall and re-install?
